src/service/requests/login/useLoginRequest.ts

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const useLogin = () => {
```

Comment:
Muda o nome do hook:
```suggestion
const useLoginRequest = () => {
```